plugins/modules/s3_bucket.py

```diff
@@ -233,6 +233,12 @@
     name: mys3bucket
     state: present
     delete_object_ownership: true
+    
```

Comment:
It's hard to see in the Github UI but you have a tab on line 236, so the pep8 check is failing.  You can run the test locally like this: 
```
$ ansible-test sanity  --python 3.8 --test pep8 plugins/modules/s3_bucket.py
```

If there's no failures the output should just look like this:
```
Running sanity test 'pep8' with Python 3.8
```

Reply:
Oh, thank you for your explanation, and sorry for the trouble. I amended my commit.